packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -27,8 +27,10 @@ async function collectAccounts(ix, accounts, last, depth = 3) {
 
   let authorizations = []
   for (let ax of accounts) {
+    let resolve = ax.resolve
+    ax.resolve = null
```

Comment:
Any need to restore `ax.resolve` later?

Reply:
Shouldn't be as this is the only time ax.resolve is ever called (unless it gets piped through the collectAccounts recursion unintentionally which is what this bug revolves around).